docs/common_tasks.rst

```diff
@@ -176,3 +176,23 @@ Building the docs locally
      > make html
 
 * The built docs can then be found inside docs/_build/html/
+
+Sharing UI-only changes with others using GitHub Pages
+------------------------------------------------------
+
+It's possible to share UI-only changes with others (for prototyping/testing) using
+GitHub Pages. This is recommended over pushing a custom branch to stage.
+
+To do this:
+
+* Fork the Treeherder repository to your own Github account.
+
+* Make a copy of the branch you wish to test, called ``gh-pages``.
+
+* Add a commit to that branch that copies ``ui/js/config/sample.local.conf.js`` to ``ui/js/config/local.conf.js``.
```

Comment:
You may also mention that you'll need to add with a ``-f`` because the file is "ignored" normally.